Sources/Commands/SwiftTool.swift

```diff
@@ -452,42 +452,63 @@ public class SwiftTool {
         }
     }
 
-    func editablesPath() throws -> AbsolutePath {
+    private func editsDirectory() throws -> AbsolutePath {
+        // 👀 is multiroot-data-file in use?
```

Comment:
👀 @neonichu @abertelrud how is this used?

Reply:
I don't really know — I think this term "multiroot" was used for cases in which there isn't just a single package as a root, e.g. in libSwiftPM clients such as Xcode in which there can be a single workspace with a bunch of different root packages, without one of them necessarily been _the_ package at the top.  I'll poke around the code, and see if I can comment more.

Reply:
@abertelrud right that was my guess as well, the question is why is it part of SwiftTool since the integration with IDEs is done at the API level not the CLI level afaik?

Reply:
cc @aciidb0mb3r @neonichu if they historical reference to this

Reply:
There's the capability of using a workspace in `swift build` as well, I believe it was added because LSP needed it.

Reply:
I think this might be something that could be replaced with explicit overrides once we implement them, but for now it's being used by some of the projects in the toolchain.

Reply:
okay thanks.